demos/switch.html

```diff
@@ -119,47 +107,6 @@ <h2>Switch on Light Theme - Disabled</h2>
           <label for="basic-switch--disabled" class="mdc-switch-label">off/on</label>
         </div>
       </section>
-
-      <section class="example mdc-theme--dark">
-        <h2>Switch on Dark Theme</h2>
```

Comment:
Can we rename the two remaining headings?

* "Switch on Light Theme" -> "Enabled"
* "Switch on Light Theme - Disabled" -> "Disabled"